apis/sgv2-quarkus-common/CONFIGURATION.md

```diff
@@ -45,11 +45,12 @@
 ### Multi-tenancy configuration
 *Configuration mapping for multi tenant operation mode, defined by [MultiTenancyConfig.java](src/main/java/io/stargate/sgv2/api/common/config/MultiTenancyConfig.java).*
 
-| Property                                                 | Type      | Default | Description                                                                                                          |
-|----------------------------------------------------------|-----------|---------|----------------------------------------------------------------------------------------------------------------------|
-| `stargate.multi-tenancy.enabled`                         | `boolean` | `false` | If multi-tenancy operation mode is enabled.                                                                          |
-| `stargate.multi-tenancy.tenant-resolver.type`            | `String`  | unset   | Tenant identifier resolver type if multi-tenancy is enabled. Possible options are `subdomain`, `fixed`, or `custom`. |
-| `stargate.multi-tenancy.tenant-resolver.fixed.tenant-id` | `String`  | unset   | Tenant identifier value if the `fixed` type is used.                                                                 |
+| Property                                                     | Type      | Default | Description                                                                                                          |
+|--------------------------------------------------------------|-----------|---------|----------------------------------------------------------------------------------------------------------------------|
+| `stargate.multi-tenancy.enabled`                             | `boolean` | `false` | If multi-tenancy operation mode is enabled.                                                                          |
+| `stargate.multi-tenancy.tenant-resolver.type`                | `String`  | unset   | Tenant identifier resolver type if multi-tenancy is enabled. Possible options are `subdomain`, `fixed`, or `custom`. |
+| `stargate.multi-tenancy.tenant-resolver.fixed.tenant-id`     | `String`  | unset   | Tenant identifier value if the `fixed` type is used.                                                                 |
+| `stargate.multi-tenancy.tenant-resolver.subdomain.max-chars` | `String`  | unset   | Optional, takes only a defined number of max characters from the resolved sub-domain as tenant id.                   |
```

Comment:
I'm fine with whatever name, this could alternatively be "prefix-length" (or "max-prefix" or "max-tenant-chars") to indicate it is the first N characters that are used.

Reply:
I am not super happy with the `max-chars`, but all of the suggestions don't feel better either.. So I am keeping this, there is a clear description in the `CONFIGURATION.MD`..